src/providers/manualInput.ts

```diff
@@ -0,0 +1,51 @@
+import inquirer from 'inquirer';
+import { ApiProvider, ProviderResponse } from '../types';
+
+export class ManualInputProvider implements ApiProvider {
+  private config: { multiline?: boolean };
+
+  constructor(config: { multiline?: boolean } = {}) {
+    this.config = config;
+  }
+
+  id() {
+    return 'manual-input';
```

Comment:
nit: `promptfoo:manual-input` or convert to a static method and set `} else if (providerPath === ManualInputProvider.id()) {` within `providers.ts:353` in order to enforce consistency.